### DISCUSSION TITLE
speed control not working after update to black 10

### DISCUSSION THREAD

**Discussion by fpjl** [gopro/labs#330] [Q&A]
After I updated the camera firmware on my black 10  my go pro labs disappeared. i reinstalled and now the speed control doesn't work. On the back where it displays the current speed it usually just stays at 0. I have uninstalled and reinstalled without success. Any suggestions? Should I wipe out everything and start again and if so how do I do that?

**Comment by dnewman-gpsw** [comment]
Are you talking about the GPS speed trigger https://gopro.github.io/labs/control/speedtrigger/  ?

I just tried that with 1.50.70 and it worked. So I will need more details.
